_pages/en/strategy-build-vs-buy.md

```diff
@@ -87,125 +63,136 @@ All ESDC personnel involved in the exploration of IT Solutions and in IT investm
+2. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (to be created - see [Coherent set of actions](#coherent-set-of-actions) and read [Managing the Risks of Obtaining of Software](#managing-the-risks-of-obtaining-of-software)).
+3. IT Product / IT Solution maintenance and support may be done in-house or outsourced to a 3rd party vendor
+   - Outsourced maintenance and support may be given to a 3rd party different than the one providing the software licence.
 
 ### Finance
 
-1. IT Solutions financial requirements are defined by the Operational Expense vs Capital Expense Framework
+1. IT Solutions financial requirements are aligned to Product Management Framework (to be created - see [Coherent set of actions](#coherent-set-of-actions))
 
 ### Legal
 
-1. IT Solutions use the ESDC Software Legal Terms Guidance
-2. IT Solutions and IT Products comply with the ESDC OSS Framework
+1. IT Solutions use the ESDC Software Legal Terms Guidance (to be created - see [Coherent set of actions](#coherent-set-of-actions))
+2. IT Solutions and IT Products comply with the ESDC OSS Management Framework
 
 ### Architecture
 
 1. IT Solutions and IT products loose coupling boundaries are at the Business Capability Model level.
    - Duplication of IT Products or IT Solutions is supported as long as they are for different business capabilities.
-2. IT Products are risk assessed for Lock-In situations with the IT Products risk assessment framework
+2. IT Products are risk assessed for lock-in situations with the IT Products risk assessment framework (to be created - see [Coherent set of actions](#coherent-set-of-actions))
 
 ## Coherent set of actions
 
-{% include table.html table-id="t001" element-id="bvb-actions" file="build-vs-buy"%}
+{% include table.html table-id="t001" file="build-vs-buy"%}
 
 ## Measuring the Strategy's success
 
-[Upcoming]
+This Strategy's success will be measured by comparing the following metrics against the key elements of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and [on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604).
+
+- Adoption/Building
+  - Increase adoption of open source software components
+  - Increase contributions to third party OSS projects
+  - Decrease number of forked versions of OSS unaligned with upstream or without contributions of changes
+  - Increase number of internal projects released as OSS
+  - Increase external contributions to ESDC OSS projects
+- Buy
+  - Decrease lock-in situations (need to define how measure)
+  - Decrease use of non-interoperable and exclusive:
+    - file formats
+    - APIs
+    - data-exports
+  - Increase transfer of systems runtime to third party Cloud service providers (Cloud First Strategy)
```

Comment:
Do we need something about increased speed and flexibility for changes? This would apply for either build/adopt or buy?  Or increased access to source code (even if not open source? e.g. we a 3rd party company builds something for us and gives us the code)?

Reply:
Shouldnt this be something like "increase number of workloads migrated to cloud environments" not neccessarily "increase transfer" which I interpret as saying "increase the rate of systems being transfered"?